### PR TITLE
Users

### PR DESCRIPTION
**Pull request by JRJurman**
change search to room to represent the state of the room
using different shell executor to fix zombie processes
added login button, well, actions, and reducer
fixed voting logic to force users to vote for movies once
added padding to make edges look better on narrower screens

lots of indentation fixes too

### CODE REVIEW

File: src/client/js/components/main-page.jsx [hunk 1]
```diff
@@ -46,19 +46,40 @@ export const MainPage = React.createClass({
...
     return (
       <div>
-        <div style={{display:'flex', alignItems:'center'}}>
-          <div className="title" style={{flex:'1 70%'}}>
+        <div style={{
+          display:'flex',
+          alignItems:'center',
+          justifyContent:'space-between'
+        }}>
+          <div className="title" style={{}}>
```
Comment (chtinahow) [inline src/client/js/components/main-page.jsx]: This is ugly, fix this old artifact

File: src/client/js/reducers/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+import user from './user';
 
 export default combineReducers({
-	search,
-	movies
-}); 
\ No newline at end of file
+	room,
+	movies,
+  user
```
Comment (chtinahow) [inline src/client/js/reducers/index.js]: Why is user on its own level?

File: src/client/js/reducers/room.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import {  EXPAND_SEARCH, COLLAPSE_SEARCH,
+          EXPAND_LOGIN, COLLAPSE_LOGIN } from '../actions/room-actions';
+
+export default function room(state = {
+  isSearchExpanded: false,
+  isLoginExpanded: false
+}, action) {
+  switch(action.type){
```
Comment (chtinahow) [inline src/client/js/reducers/room.js]: Indentation poor favour

**Comment by JRJurman** [comment]
tina said "fine"

**Pull request merged**